portfolio/src/main/webapp/script.js

```diff
@@ -1,5 +1,24 @@
+/** Fetch quote from server and add to DOM */
 function getRandomQuote(){
-    fetch('/data').then(response => response.text()).then((quote) => {
+    fetch('/quotes').then(response => response.text()).then((quote) => {
         document.getElementById('quote-container').innerText = quote;
     })
+}
+
+/** Fetch comments from server and add to DOM */
+function getComments(){
+    fetch('/data').then(response => response.json()).then((messages) => {
+        const commentElement = document.getElementById('video-comments');
+        commentElement.innerHTML = '';
+        commentElement.appendChild(createListElement(messages[0]));
```

Comment:
While currently you are card coding in 3 comments, can you come up with a more dynamic way to do this?

You can use a for-loop on the messages array which will also remove the multiple lines containing createListElement and appendChild :)